.github/workflows/ci-weekly.yml

```diff
@@ -1,74 +1,164 @@
-name: CI - FAST Validation
+name: CI - FAST
```

Comment:
Let's keep the name.  It's important to distinguish this from the others.

Reply:
Please update this PR with description and it looks like the check boxes in that detail are misformatted as well.

Reply:
I just change the tittle